ema_workbench/em_framework/optimization.py

```diff
@@ -485,10 +632,18 @@ class HyperVolume(AbstractConvergenceMetric):
     minimum : numpy array
     maximum : numpy array
 
+
+    This class is deprecated. Use ArchiveLogger instead and calculate hypervolume
+    in post using HypervolumeMetric as also shown in the directed search tutorial.
```

Comment:
Is the use of this class removed from all the examples? Otherwise those will start generating warnings